### PR TITLE
updated sign in options on full page view to include more login options

### PR DESCRIPTION
**Pull request by leewynne**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

This PR request adds 'more sign in options' on the full desktop view

## Related Tickets & Documents

#9346 

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text
Screenshot 2020-07-20 at 11 48 05

### CODE REVIEW

File: app/views/shared/authentication/_providers_sidebar.html.erb [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
-    data-no-instant>
-    <%= inline_svg_tag("#{provider.provider_name}.svg", aria: true, class: "crayons-icon", title: provider.provider_name) %>
-    Sign In with <%= provider.official_name %>
-  </a>
-<% end %>
+<div class="border-0 border-b-1 border-solid border-base-20 p-2 grid gap-2">
+  <% if any_selfserve_auth? %>
+    <%= render partial: "shared/authentication/providers_nav_menu" %>
+    <a
+      href="/enter"
```
Comment (citizen428) [inline app/views/shared/authentication/_providers_sidebar.html.erb:5]: **nitpick:** We're attempting to use path helpers as much as possible, so let's please do this here too.

```suggestion
      href="<%= sign_up_path %>"
```

File: app/views/shared/authentication/_providers_sidebar.html.erb [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+      class="crayons-btn crayons-btn--secondary">
+      More Sign In Options
+    </a>
+  <% else %>
+    <a
+      href="/enter"
```
Comment (citizen428) [inline app/views/shared/authentication/_providers_sidebar.html.erb:11]: **nitpick:** Same as above.

```suggestion
      href="<%= sign_up_path %>"
```

**Comment by leewynne** [comment]
Ooooo that's good to know 😊

**Comment by mstruve** [comment]
Hey @leewynne! This ended up being fixed in https://github.com/forem/forem/pull/9496 and then we overhauled it to add the email option just to the sidebar. I think this covers this issue and PR so I am going to close it. 

Screen Shot 2020-09-18 at 10 08 02 AM

**Pull request closed without merging**